src/python/pants/backend/helm/goals/lint.py

```diff
@@ -32,7 +32,7 @@ class HelmLintFieldSet(HelmChartFieldSet):
 
 class HelmLintRequest(LintTargetsRequest):
     field_set_type = HelmLintFieldSet
-    tool_name = HelmSubsystem.options_scope
+    tool_subsystem = HelmSubsystem  # type: ignore[assignment]
```

Comment:
why does this need a type ignore, and not the others?

Reply:
Doesn't have a `skip` option........ yet :smile:

Reply:
Oohh.. ok.